README.md

```diff
@@ -1765,6 +1765,14 @@ This library does not support comments by default. It does so for three reasons:
 
 However, you can pass set parameter `ignore_comments` to true in the `parse` function to ignore `//` or `/* */` comments. Comments will then be treated as whitespace.
 
+### Trailing Commas
+
+Trailing commas in arrays and objects are alse not part of the [JSON specification](https://tools.ietf.org/html/rfc8259), and this library does not support it by default.
+
+Like comments, you can pass set parameter `allow_trailing_commas` to true in the `parse` function to allow trailing commas in arrays and objects. Trailing commas will then be ignored.
```

Comment:
If this is added, should this be called `ignore_trailing_commas` to match `ignore_comments`?

Reply:
I choose "allow" because I think that commas are not like comments: successive commas are invalid (`[1,,]`), but successive comments are valid (`[1 /*, 2*/ /*, 3*/]`).

Do you think `ignore_trailing_commas` is better? If you really think so, I would rename it, or I would try to avoid using "ignore" when describing trailing commas.

Reply:
I don't see a difference in meaning between `ignore` and `allow`, and so this looks to me like a needless difference, but @nlohmann has the final say here.